tethysapp/gwdm/templates/gwdm/base.html

```diff
@@ -34,13 +34,13 @@
 {% url 'gwdm:upload-rasters' as upload_rasters %}
 {% url 'gwdm:delete-rasters' as delete_rasters %}
 <br>
-<li class="{% if request.path == home_url %}active{% endif %}"><a href="{{home_url}}">Home</a></li>
-<li class="{% if request.path == metrics_url %}active{% endif %}"><a href="{{metrics_url}}">Summary Statistics</a></li>
+<li class="nav-item{% if request.path == home_url %} active{% endif %}"><a class="nav-link" href="{{home_url}}">Home</a></li>
```

Comment:
The `active` class should be moved to the `<a>` tag.

For example, this:

```
<li class="nav-item{% if request.path == home_url %} active{% endif %}"><a class="nav-link" href="{{home_url}}">Home</a></li>
```
should be changed to this:

```
<li class="nav-item"><a class="nav-link{% if request.path == home_url %} active{% endif %}" href="{{home_url}}">Home</a></li>
```

Fix all cases in this file.